static/js/tools/timeline/chart.tsx

```diff
@@ -135,8 +135,15 @@ class Chart extends Component<ChartPropsType, ChartStateType> {
     const placeName = Object.values(this.props.placeNames)[0];
     const deltaCheckboxId = `delta-cb-${this.props.mprop}`;
     const sourceSelectorSvInfoList = this.getSourceSelectorSvInfo(statVars);
+    // Hides empty plots
```

Comment:
we'd prefer to display an error message in this case, rather than hiding the entire chart.

Reply:
How should the error be rendered, do you have pointers? Is it actually an error, or is it expected that some StatVars do not have data? An example of this is selecting Increment Population in the timeline tool. Perhaps we can move to separate PR

Reply:
it should be an error, at least in the timeline tool, as users should not be able to select variables which we don't have data for.

a separate PR would be great. i was thinking of doing something similar in this case too, when it came up in another context.

Reply:
I undid the changes in this file.